### PR TITLE
Remove Replex Code

### PR DESCRIPTION
**Pull request by Maithem**
Removed replex code.

### CODE REVIEW

File: runtime/src/main/java/org/corfudb/protocols/wireprotocol/WriteRequest.java [hunk 1]
```diff
@@ -22,44 +22,34 @@
     @SuppressWarnings("unchecked")
     public WriteRequest(ByteBuf buf) {
         writeMode = ICorfuPayload.fromBuffer(buf, WriteMode.class);
-        if (writeMode == WriteMode.REPLEX_STREAM) {
-            streamAddresses = ICorfuPayload.mapFromBuffer(buf, UUID.class, Long.class);
-        } else { streamAddresses = null; }
+        streamAddresses = null;
```
Comment (no2chem) [inline runtime/src/main/java/org/corfudb/protocols/wireprotocol/WriteRequest.java]: I think we can remove the streamAddresses field

File: infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java [hunk 2]
```diff
@@ -148,22 +148,11 @@ public void write(CorfuPayloadMsg<WriteRequest> msg, ChannelHandlerContext ctx,
         // clear any commit record (or set initially to false).
         msg.getPayload().clearCommit();
```
Comment (no2chem) [inline infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java]: I think we can also remove the commit record

**Comment by corfudb-performance** [comment]
Results automatically generated by _CorfuDB Benchmark Framework_ to assess the performance of this pull request for commit **837d23b**.

*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 10 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 10 threads, Disk mode
 
 An interactive dashboard with Pull Request Performance Metrics for ALL cluster types and numbers of threads in run, is available at:
 [Pull Request #616 Graphs ](https://snapshot.raintank.io/dashboard/snapshot/hTI0ieaVJBYGfwt4qOnGDinWFlLmz1wL)

**Comment by corfudb-performance** [comment]
Results automatically generated by _CorfuDB Benchmark Framework_ to assess the performance of this pull request for commit **4ef307d**.

*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 10 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 10 threads, Disk mode
 
 An interactive dashboard with Pull Request Performance Metrics for ALL cluster types and numbers of threads in run, is available at:
 [Pull Request #616 Graphs ](https://snapshot.raintank.io/dashboard/snapshot/W3qqpVODmN8G1ms842OJVzWMNN2HfRDR)

**Pull request merged**